src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/enable-serverless-monitoring-aws-lambda-layerless.mdx

```diff
@@ -204,10 +228,16 @@ If you're using a Lambda function without a layer, you'll need to instrument New
 
     9. Optional: If your Lambda function is using ES Modules, set the environment variable `NEW_RELIC_USE_ESM` to `true`. Note that if using ES Modules, you must use async/await or promises for your function. Callback-based functions are not supported.
 
-    10. Invoke the Lambda at least once. This creates a CloudWatch log group, which must be present for the next step to work.
+    10. To complete the instrumentation, follow one of the following steps to send the telemetry data to New Relic:
+          - Use the New Relic Lambda `Extension layer`. You can get the latest [(NewRelicLambdaExtension)](https://layers.newrelic-external.com/) layer arn. You can either use AWS CLI to install the layer or manually add it to your Lambda. 
```

Comment:
Suggestion: use "*" instead of "-" for bulleted list.
 example:  * Sample Code for CommonJS: